psh/test.yaml

```diff
@@ -41,6 +43,7 @@ test:
               exclude:
                   - armv7m7-imxrt106x
                   - armv7m7-imxrt117x
+                  - armv7a9-zynq7000-qemu
```

Comment:
@HBuczynski I assume it's not yet possible to create dirs on zynq-qemu?

Reply:
Creating dirs works fine (test ls uses this functionality), but there may be other problems, like those related to ownership.
![Screenshot from 2022-04-25 19-01-53](https://user-images.githubusercontent.com/77304431/165138517-8119d63a-7f71-4888-ac82-4e478c602aca.png)
:

Reply:
I confirm that `mkdir` works fine on `zynq-qemu`.

Reply:
ok, so should pass but doesn't - seems like a bug? (either in the target implementation or in test) - we should fix it in instead of not running the test :).

Reply:
Apart from disabling checking ownership in the test (not needed for now, no multi-user support), I had to make one file name shorter to enable the test for all targets.